app/components/collections/workflow_review_component.rb

```diff
@@ -19,7 +19,7 @@ def review_workflow_status
 
     sig { returns(T.nilable(String)) }
     def reviewers
-      collection.reviewers.map(&:sunetid).join(', ')
+      collection.reviewed_by.map(&:sunetid).join(', ')
```

Comment:
Can you say more about this change?

Reply:
Sure: https://github.com/rails/rails/issues/41010